README.md

```diff
@@ -17,30 +18,67 @@ All contracts are located in the `contracts` folder.
     - [Modifiers](#forkdelta-modifiers)
     - [Functions](#forkdelta-functions)
 
+
+## Development
+
+### Setting up a development environment
+Requirements:
+* Node.js (https://nodejs.org/en/)
+* git client (https://git-scm.com/download/)
+
+Setup:
+1. Install truffle: `npm install -g truffle`
+2. On Windows install the necessary build tools: `npm install --global --production windows-build-tools`
+3. Install Ganache from: http://truffleframework.com/ganache/
+4. Clone the repo: git clone https://github.com/forkdelta/smart_contract.git
+5. Change into the root directory: `cd smart_contract`
+6. Install all node.js requirements from package.json: `npm install`
+7. If you use VSCode, copy `.vscode\settings.json.default` to `.vscode\settings.json` for a reasonable solhint linter configuration
+
+### Migrating and testing with Ganache:
+* Start ganache
+* Compile: `truffle compile`
+* Migrate: `truffle migrate`
+* Run test cases: `truffle test`
+
+### Migrate and test with truffle develop
```

Comment:
+### Migrating and testing with Ganache:
+### Migrate and test with truffle develop

Let's stick with one style of formatting for these and past/present tense

Reply:
Fixed with the next commit.